core/src/sumeragi/message.rs

```diff
@@ -57,19 +58,21 @@ impl From<ValidBlock> for BlockCreated {
 #[non_exhaustive]
 pub struct BlockSigned {
     /// Hash of the block being signed.
-    pub hash: HashOf<BlockPayload>,
-    /// Set of signatures.
-    pub signatures: SignaturesOf<BlockPayload>,
+    pub hash: HashOf<SignedBlock>,
```

Comment:
Why it's now `HashOf<SignedBlock>`? 
`BlockSignature` itself is `SigantureOf<BlockPayload>`

Reply:
`HashOf<SignedBlock>` does not serve as block identifier during consensus as we recently discussed in #4641

Reply:
yes, I used `hash_of_payload` now